Automattic/jetpack
# Connection Package: Remove src/js files from final bundle

We were shipping JavaScript source files for the Connection package, thus adding about 8k to plugins.

#### Changes proposed in this Pull Request:

* Updates `.gitattributes` in the Connection package to not include `src/js` files

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Wait for the build check to finish, click details, then summary, search for the artifacts at the bottom of the page download the plugins.zip file and confirm the src/js dir is not included inside jetpack-dev/jetpack_vendor/automattic/jetpack-connection/src
